pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -480,7 +481,7 @@ async def _get_event_iterator(self) -> AsyncIterator[ModelResponseStreamEvent]:
                 current_block = event.content_block
                 if isinstance(current_block, BetaTextBlock) and current_block.text:
                     maybe_event = self._parts_manager.handle_text_delta(
-                        vendor_part_id='content', content=current_block.text
+                        vendor_part_id='content', content=current_block.text, model_profile=self._model_profile
```

Comment:
If we're not passing `extract_think_tags=True` yet, we don't need to pass think tags or profile either -- that means the thinking parts will always come in separate API fields, not embedded in the text.

So we can remove the changes from all the files that don't already use `extract_think_tags=True`.